Makefile

```diff
@@ -20,8 +20,8 @@ TAG_LATEST ?= false
 # LOCALIP as an env var before running 'make local' will solve that.
 LOCALIP ?= $(shell ifconfig | grep inet | grep -v '::' | grep -v 127.0.0.1 | head -n1 | awk '{print $$2}')
 
-# Sets GIT_REF to a tag if it's present, otherwise the short rev.
-GIT_REF = $(shell git describe --tags || git rev-parse --short=8 --verify HEAD)
+# Sets GIT_REF to a tag if it's present, otherwise the basename of a git describe is used.
+GIT_REF = $(shell git describe --tags --exact-match 2>/dev/null || basename $(shell git describe --all --long))
```

Comment:
As per @stevesloka , let's revert the `git describe` to `git rev-parse --short=8 --verify HEAD`. Since you added the `--exact-match` flag, we will hit this case now, and the `GIT_BRANCH` variable takes care of the branch field of the long description.